photon-lib/src/main/scala/com/linkedin/photon/ml/hyperparameter/estimators/kernels/StationaryKernel.scala

```diff
@@ -65,11 +81,12 @@ abstract class StationaryKernel(
   override def apply(x: DenseMatrix[Double]): DenseMatrix[Double] = {
     require(x.rows > 0 && x.cols > 0, "Empty input.")
 
-    val ls = expandDimensions(lengthScale, x.cols)
-    val dists = pairwiseDistances(x(*,::) / ls)
+    val u = transform(x)
```

Comment:
Something like `xTransformed` would be a little more readable than `u`. Same for `x1Transformed`, `x2Transformed` below.